Gems/AudioSystem/Code/Platform/Common/Default/AudioSystemGemSystemComponent_default.cpp

```diff
@@ -14,41 +14,14 @@ namespace Audio::Platform
 {
     void InitializeAudioAllocators()
     {
-        // Create audio system memory pool
         if (!AZ::AllocatorInstance<AudioSystemAllocator>::IsReady())
         {
-            const size_t heapSize = Audio::CVars::s_ATLMemorySize << 10;
-
-            AudioSystemAllocator::Descriptor allocDesc;
-
-            // Generic Allocator:
-            allocDesc.m_allocationRecords = true;
-            allocDesc.m_heap.m_numFixedMemoryBlocks = 1;
-            allocDesc.m_heap.m_fixedMemoryBlocksByteSize[0] = heapSize;
-
-            allocDesc.m_heap.m_fixedMemoryBlocks[0] = AZ::AllocatorInstance<AZ::OSAllocator>::Get().Allocate(
-                allocDesc.m_heap.m_fixedMemoryBlocksByteSize[0],
-                allocDesc.m_heap.m_memoryBlockAlignment
-            );
-
-            AZ::AllocatorInstance<AudioSystemAllocator>::Create(allocDesc);
+            AZ::AllocatorInstance<AudioSystemAllocator>::Create();
```

Comment:
Is there no longer a way to make a fixed-size block for an allocator to use?  Sometimes platforms have strict memory budgets, how do we express that (i.e. keep these cvars meaningful)?